src/main/java/com/group1/taskmanagement/model/User.java

```diff
@@ -23,6 +23,12 @@ public class User {
     @Column(nullable = false, unique = true)
     private String username;
 
+    @Column(nullable = true, unique = false)
+    private String firstname;
+
+    @Column(nullable = true, unique = false)
+    private String lastname;
```

Comment:
Can you camelCase these variable names everywhere? The rest of the app is using camelCase. That way we don't run into any casing errors on the front end from inconsistency. 

Also, you don't need a @Column with the nullable and unique declaration. Those are the default values for @Column.